### DISCUSSION TITLE
E2E encrypted Origin<-->Browser with TLS Passthrough at Gateway

### DISCUSSION THREAD

**Discussion by Winterhuman** [ipns-link/specs#3] [Ideas]
# Draft
https://github.com/ipns-link/specs/blob/main/trustless.md

**Comment by SomajitDey** [comment]
@ipns-link/devs Thoughts? https://github.com/ipns-link/specs/blob/main/trustless.md

**Reply by Winterhuman** [reply]
Looks good, I think it's a solid plan.
